components/brave_new_tab_ui/containers/newTab/settings/cards.tsx

```diff
@@ -84,6 +89,18 @@ function CardSettings({ toggleShowBraveTalk, showBraveTalk, braveTalkSupported,
       </StyledSettingsInfo>
       <ToggleButton on={showRewards} toggleFunc={toggleShowRewards} />
     </SettingsWidget>}
+    {loadTimeData.getBoolean('vpnWidgetSupported') && <SettingsWidget>
+      <StyledBannerImage src={braveVPNBanner} />
+      <StyledSettingsInfo>
+        <StyledSettingsTitle>
+          {getLocale('braveVpnWidgetSettingTitle')}
+        </StyledSettingsTitle>
+        <StyledSettingsCopy>
+          {getLocale('braveVpnWidgetSettingDesc')}
+        </StyledSettingsCopy>
+      </StyledSettingsInfo>
+      <ToggleButton on={!!showBraveVPN} toggleFunc={() => saveShowBraveVPN(!showBraveVPN)} />
```

Comment:
showBraveVPN is a boolean right, so do we need the `!!`?


```suggestion
      <ToggleButton on={showBraveVPN} toggleFunc={() => saveShowBraveVPN(!showBraveVPN)} />
```

Reply:
ah yeah, then carry on :)